src/main/scala/firrtl/passes/memlib/ReplaceMemTransform.scala

```diff
@@ -96,8 +96,9 @@ Optional Arguments:
 class SimpleTransform(p: Pass, form: CircuitForm) extends Transform {
   def inputForm = form
   def outputForm = form
-  def execute(state: CircuitState): CircuitState = state.copy(circuit = p.run(state.circuit))
+  def execute(state: CircuitState): CircuitState = state.copy(circuit = p.run(state.circuit), annotations = None)
```

Comment:
The rename map is generally not passed to a new transform, so state.renamemap should always be none. But you are right, we should probably play it safe and just return CircuitState(circuit, form)